### ISSUE TITLE
Increase API version to 1.0

### ISSUE THREAD

**Issue by ffdixon** [bigbluebutton/bigbluebutton#2995]
Currently, the returned value for

  http://demo.bigbluebutton.org/bigbluebutton/api

is

```
<response>
<returncode>SUCCESS</returncode>
<version>0.9</version>
</response>
```

Need to update the version number to 1.0

**Comment by GhaziTriki** [comment]
@ritzalam there is a file to bump the whole BBB version https://github.com/bigbluebutton/bigbluebutton/blob/master/scripts/bump_version.sh if you can add the file where API version is changed to it

**Comment by ffdixon** [comment]
Fixed in latest builds of BigBlueButton 1.0-beta.  This will be pushed shortly to the public repository.
